templates/partials/applicantPasswordModal.phtml

```diff
@@ -0,0 +1,26 @@
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
```

Comment:
is there a reason this is here instead of the view? also, can we not use the CDN